pyshop/models.py

```diff
@@ -167,8 +166,13 @@ def name(self):
     def _get_password(self):
         return self._password
 
+    @classmethod
+    def hash_password(cls, pw):
+        return bcrypt.hashpw(pw.encode('utf8'), bcrypt.gensalt(12))
+
     def _set_password(self, password):
-        self._password = unicode(crypt.encode(password))
+        # we want to store this in the database as unicode
+        self._password = unicode(hashpw(password))
```

Comment:
nit: using `unicode(obj)` to go from bytes to unicode is unsafe (and results in calling `repr` if you do the equivalent in python 3000) and relies on `$LOCALE` (so could do weird things in weird locals); prefer `obj.decode('ascii')` if that is what you want

Reply:
https://github.com/EasyPost/pyshop/blob/master/pyshop/compat.py#L6

I think this existing monkeypatch is probably even worse but it's used in a lot of places.  If I decide to move forward with py3 I'll just take this out.